package.json

```diff
@@ -9,7 +9,8 @@
   "main": "build/index.js",
   "types": "build/index.d.ts",
   "dependencies": {
-    "js-cookie": "^2.2.1"
+    "js-cookie": "^2.2.1",
+    "prop-types": "^15.7.2"
```

Comment:
I added this as suggested in the official docs https://www.npmjs.com/package/prop-types#how-to-depend-on-this-package